static/css/timeline_statsvar.scss

```diff
@@ -0,0 +1,220 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "tools/base_tools";
+
+$menu-background: #525252;
```

Comment:
This looks similar to this file:
https://github.com/datacommonsorg/website/blob/master/static/css/tools/statvar_menu.scss

If you'd like to componentize the search css from timeline.scss, you can move those out.

I'm not sure why you're reverting back to the old styles though? Can you please keep the new look and feel?

Reply:
Ah sorry, it looks like you've maybe misunderstood my last comment. What I meant was:

- create new dev_menu.scss
- in that file, @include base_tools and statvar_menu
- add dev_menu.scss to the dev_menu entry in webpack.config.json

Reply:
Got it. Sorry for the misunderstanding.